pennylane/ops/channel.py

```diff
@@ -51,13 +51,10 @@ class AmplitudeDamping(Channel):
         gamma (float): amplitude damping probability
         wires (Sequence[int] or int): the wire the channel acts on
     """
+    num_params = 1
```

Comment:
```suggestion

    num_params = 1
    """int: Number of trainable parameters that this operator depends on."""
    
```

Sorry to do this, but if it is simple enough, could you include this docstring? I just realised that otherwise, the gates with class attribute do not have a nice documentation. 

I will try to add this for all previous changes in the op-refactor branch, but it would help me to not miss these when the PR gets merged :)

Reply:
Currently num_params and num_wires are documented in the main docstring of the class:
```
**Details:**

    * Number of wires: 1
    * Number of parameters: 1
```
Do you want to get rid of this? Do you then want to have a separate docstring for `num_wires` as well?

Reply:
As discussed elsewhere, ignore my remark for now :)